src/data.tsx

```diff
@@ -0,0 +1,5 @@
+export const languages = [
```

Comment:
be specific in naming

Reply:
As per Mika Advise nothing to resolve to this issue